.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - openjdk8
+  - openjdk9
```

Comment:
Guess we must keep java 8 there.
Between that and disabling  a buggy plugin (animal sniffer) guess last one is saner.